vesc_ackermann/src/ackermann_to_vesc.cpp

```diff
@@ -37,28 +37,22 @@ namespace vesc_ackermann
 {
 
 template <typename T>
-inline bool getRequiredParam(const ros::NodeHandle& nh, const std::string& name, T* value);
+inline bool getRequiredParam(const ros::NodeHandle& nh, std::string name, T& value);
 
 AckermannToVesc::AckermannToVesc(ros::NodeHandle nh, ros::NodeHandle private_nh)
 {
   // get conversion parameters
-  if (!getRequiredParam(nh, "speed_to_erpm_gain", &speed_to_erpm_gain_))
+  if (!getRequiredParam(nh, "speed_to_erpm_gain", speed_to_erpm_gain_))
     return;
-  if (!getRequiredParam(nh, "speed_to_erpm_offset", &speed_to_erpm_offset_))
+  if (!getRequiredParam(nh, "speed_to_erpm_offset", speed_to_erpm_offset_))
     return;
-  if (!getRequiredParam(nh, "accel_to_current_gain", &accel_to_current_gain_))
+  if (!getRequiredParam(nh, "steering_angle_to_servo_gain", steering_to_servo_gain_))
     return;
-  if (!getRequiredParam(nh, "accel_to_brake_gain", &accel_to_brake_gain_))
-    return;
-  if (!getRequiredParam(nh, "steering_angle_to_servo_gain", &steering_to_servo_gain_))
-    return;
-  if (!getRequiredParam(nh, "steering_angle_to_servo_offset", &steering_to_servo_offset_))
+  if (!getRequiredParam(nh, "steering_angle_to_servo_offset", steering_to_servo_offset_))
```

Comment:
Several of these changes are breaking changes (removing existing parameters and renaming others). This means a new major version would have to be released. Not that I won't do it, just giving you a heads-up.

Reply:
I will take a closer look and run some fixes for the noetic branch! 
Thanks for the feedback! Very helpful to have another set of eyes on the code!